lib/sass/form/components/Combobox2/sass/Combobox.scss

```diff
@@ -10,7 +10,7 @@
 
   &.disabled {
     .tyk-form-control {
-      border: $form-control-disabled-border;
+      border: $form-control-disabled-border !important;
```

Comment:
why the important here? why can't you manage this in your code?

Reply:
it needs to trump this piece of css which uses `!important`
![tyk-form-control-focus-important](https://user-images.githubusercontent.com/5877494/212067273-fcad4263-939c-4527-b62b-5a681516914f.png)